service/src/main/java/org/ehrbase/aql/compiler/tsclient/OpenehrTerminologyServer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Vitasystems GmbH, Hannover Medical School, and Luis Marco-Ruiz (Hannover Medical School).
+ *
+ * This file is part of project EHRbase
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehrbase.aql.compiler.tsclient;
+/***
+ *@Created by Luis Marco-Ruiz on Mar 6, 2020
+ *
+ * @param <DvCodedText> concept type
+ * @param <ID> id type
```

Comment:
I'm not sure I understand the 'id type' could you give an example as part of the documentation?

Reply:
hi @chevalleyc , This is left to generics because the implementation of an ID can be a simple String (ICD 10), a number (SNOMED-CT), or more complex types. Therefore is left as a generic so in the future different TS adaptors can deal with different ways of identifying the codes.